src/humanPose/spaghetti.js

```diff
@@ -75,16 +83,28 @@ class MeasurementLabel {
 const SpaghettiSelectionState = {
     NONE: {
         onPointerDown: (spaghetti, e) => {
+            if (e.deselectedSpaghetti) {
+                // If another spaghetti was deselected on this click, do nothing
```

Comment:
This might be easier to accomplish using stopPropagation() in the deselection handler